webview/src/experiments/components/App.test.tsx

```diff
@@ -182,7 +182,12 @@ describe('App', () => {
     const headerB = screen.getByText('B')
     const headerD = screen.getByText('D')
 
-    dragAndDrop(headerB, headerD, DragEnterDirection.AUTO)
+    dragAndDrop(
+      headerB,
+      // eslint-disable-next-line testing-library/no-node-access
+      headerD.parentElement?.parentElement || headerD,
```

Comment:
[Q] Can we follow up and wrap these calls in a function so we don't have `eslint-disable` all through the tests?

```
// eslint-disable-next-line testing-library/no-node-access
      headerD.parentElement?.parentElement || headerD
```

Reply:
limit them to a single place like we try to do when using `any`